Add time series histogram to the TSDB docs. (#4548)
elastic/docs-content: manage-data/data-store/data-streams/downsampling-concepts.md

3. For each [time series metric](time-series-data-stream-tsds.md#time-series-metric) field, computes aggregations for all documents in the bucket.

    * `gauge` field type:
        * `min`, `max`, `sum`, and `value_count` are stored as type `aggregate_metric_double`.
    * `counter` field type:
        * the last value is stored and the type is preserved.

>     * `histogram` field type: {applies_to}`stack: preview 9.3`

Reviewer: add also: {applies_to}`serverless: preview`

>         * all the individual histograms are merged to a single histogram which is stored and the type is preserved. In the case of the `histogram` field type, the [T-Digest](elasticsearch://reference/aggregations/search-aggregations-metrics-percentile-aggregation.md) algorithm is used.

Reviewer: ```suggestion
        * individual histograms are merged into a single histogram which is stored and the type is preserved. In the case of the `histogram` field type, the [T-Digest](elasticsearch://reference/aggregations/search-aggregations-metrics-percentile-aggregation.md) algorithm is used.
```
